datasets/flwr_datasets/federated_dataset.py

```diff
@@ -51,6 +54,11 @@ class FederatedDataset:
         (representing the number of IID partitions that this split should be partitioned
         into). One or multiple `Partitioner` objects can be specified in that manner,
         but at most, one per split.
+    partition_division : Optional[Union[List[float], Tuple[float, ...], Dict[str,
+    float]]]
+        Divide each partition into splits (e.g. into train and validation) and
+        control the size of the splits - fractions of the data. You can also name the
+        splits for verification.
```

Comment:
Can we try to avoid the `split` terminology here to prevent people from confusing it with the `resplitter`?

Reply:
I rewrote the whole description (and also got rid of`split`)